src/main/scala/mesosphere/marathon/api/v2/EventSubscriptionsResource.scala

```diff
@@ -35,7 +36,9 @@ class EventSubscriptionsResource @Inject() (val config: MarathonConf) extends Re
   @Timed
   def subscribe(@Context req: HttpServletRequest, @QueryParam("callbackUrl") callbackUrl: String): Response = {
     validateSubscriptionService()
-    val future: Future[MarathonEvent] = service.handleSubscriptionEvent(Subscribe(req.getRemoteAddr, callbackUrl))
+    validateUrl(callbackUrl)
```

Comment:
We moved to a validation API (example: https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/state/PortDefinition.scala#L15)
Do you think you can use that as well?